VideoAPI/Testing.Common/Helper/Builders/Api/BookNewConferenceRequestBuilder.cs

```diff
@@ -25,17 +25,23 @@ public BookNewConferenceRequestBuilder(string caseName)
                 .Build();
         }
 
-        public BookNewConferenceRequestBuilder WithJudge()
+        public BookNewConferenceRequestBuilder WithJudge(string firstName = null)
         {
             var participant = Builder<ParticipantRequest>.CreateNew()
-                .With(x => x.Name = $"Automation_{Name.FullName()}")
+                .With(x => x.Name = "Automation_" + Name.First() + Faker.RandomNumber.Next())
                 .With(x => x.FirstName = $"Automation_{Name.First()}")
```

Comment:
can you match all the other sting formats? $"Automation_{Name.First()}{Faker.RandomNumber.Next()}"

Reply:
Done